app/assets/javascripts/discourse/app/components/modal/flag.hbs

```diff
@@ -68,7 +68,7 @@
 
     {{#if this.showDeleteSpammer}}
       <DButton
-        class="btn-danger"
+        class="btn-danger delete_spammer"
```

Comment:
@tgxworld Is this kosher for element targeting in tests?

Reply:
Yup this is the way :)

Reply:
should be delete-spammer though